plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -221,6 +225,58 @@ def get_manifests_directory(self):
         return os.path.join(self.parent.get_working_dir(), 'manifests')
 
 
+class PublishManifestListsStep(publish_step.UnitModelPluginStep):
+    """
+    Publish ManifestLists.
+    """
+
+    def __init__(self, redirect_data, repo_content_unit_q=None):
+        """
+        Initialize the PublishManifestsStep, setting its description and calling the super class's
+        __init__().
+
+        :param redirect_data: 
+        :type  redirect_data: dict
+        :param repo_content_unit_q: optional Q object that will be applied to the queries performed
+                                    against RepoContentUnit model
+        :type  repo_content_unit_q: mongoengine.Q
+
+        """
+        super(PublishManifestListsStep, self).__init__(step_type=constants.PUBLISH_STEP_MANIFEST_LISTS,
+                                                   model_classes=[models.ManifestList],
+                                                   repo_content_unit_q=repo_content_unit_q)
+        self.description = _('Publishing Manifest Lists.')
+        self.redirect_data = redirect_data
+
+    def process_main(self, item):
+        """
+        Link the item to the Manifest file.
+
+        :param item: The Blob to process
```

Comment:
yes, you will find a lot of this typos, because of copy-paste, i will fix that once i will get to the doc blocks